### PR TITLE
PHP: add more-examples link

### PR DESCRIPTION
**Pull request by runtu666**
**Preview**: https://deploy-preview-3712--opentelemetry.netlify.app/docs/instrumentation/php/automatic/#next-steps

### CODE REVIEW

File: content/en/docs/instrumentation/php/automatic.md [hunk 1]
```diff
@@ -226,3 +226,7 @@ commonly used PHP libraries. For the full list, see
 After you have automatic instrumentation configured for your app or service, you
 might want to add [manual instrumentation](../manual) to collect custom
 telemetry data.
+
+
+## More examples
+You can find more examples and documents here [PHP examples](https://github.com/open-telemetry/opentelemetry-php-contrib)
```
Comment (theletterf) [inline content/en/docs/instrumentation/php/automatic.md:232]:
```suggestion
## Examples

You can find more examples in the [opentelemetry-php-contrib](https://github.com/open-telemetry/opentelemetry-php-contrib) repository.
```

**Review by theletterf — CHANGES_REQUESTED** [review]

**Comment by theletterf** [comment]
Thanks! Made a suggestion.

**Review by brettmc — APPROVED** [review]

**Comment by svrnm** [comment]
thanks @runtu666 , please fix the linter issues.

Run npx gulp lint-md and npm run fix:format for that

File: content/en/docs/instrumentation/php/automatic.md [hunk 2]
```diff
@@ -226,3 +226,8 @@ commonly used PHP libraries. For the full list, see
 After you have automatic instrumentation configured for your app or service, you
 might want to add [manual instrumentation](../manual) to collect custom
 telemetry data.
+
+
+## Examples
+
+You can find more examples in the [opentelemetry-php-contrib](https://github.com/open-telemetry/opentelemetry-php-contrib) repository.
```
Comment (chalin) [inline content/en/docs/instrumentation/php/automatic.md:233]: Hmm, it doesn't make sense to have a section head after "Next steps" IMHO, how about just adding the sentence to the existing last section, something like this:

```suggestion

For more examples, see
[opentelemetry-php-contrib/examples](https://github.com/open-telemetry/opentelemetry-php-contrib/tree/main/examples).
```

**Review by chalin — CHANGES_REQUESTED** [review]

**Comment by runtu666** [comment]
> thanks @runtu666 , please fix the linter issues.
> 
> Run npx gulp lint-md and npm run fix:format for that

done @svrnm

**Review by chalin — APPROVED** [review]
👍🏻

**Comment by svrnm** [comment]
@theletterf please re-review if your changes have been applied and you're satisfied now

**Review by theletterf — APPROVED** [review]

**Pull request merged**